test/e2e/monitoring_stack_controller_test.go

```diff
@@ -983,16 +984,27 @@ func newPrometheusPod(t *testing.T, name, ns string) *corev1.Pod {
 			Namespace: ns,
 			Labels: map[string]string{
 				"app.kubernetes.io/name":    "prometheus",
-				"app.kubernetes.io/version": "2.39.1",
+				"app.kubernetes.io/version": "multiarch-v0.1.0",
 			},
 		},
 		Spec: corev1.PodSpec{
 			Containers: []corev1.Container{{
-				Name:  "prometheus",
-				Image: "quay.io/prometheus/prometheus:v2.39.1",
+				Name:  "prometheus-example-app",
+				Image: "quay.io/openshifttest/prometheus-example-app@sha256:382dc349f82d730b834515e402b48a9c7e2965d0efbc42388bd254f424f6193e",
```

Comment:
I wonder if we should instead rely on what upstream Prometheus Operator is using ?

Reply:
I would prefer to use this one, for I run the upstream test cases on OCP with all kinds of profiles share between QE team during ObO release, the image will be mirrored by default and the test cases can get the image and  run on disconnected or private OCP clusters, other wise, the case will failed on private and disconnected OCP cluster.

Reply:
@lihongyan1 would you mind leaving a comment that describes this and a link to the upstream that is mirrored? Just for future us when someone asks why we don't use the upstream version :)

Reply:
Added comments